src/components/data/listing/Listing.js

```diff
@@ -240,6 +252,20 @@ function Listing(props) {
         setDetailsEnabled(selected && selected.length === 1);
     }, [selected]);
 
+    useEffect(() => {
+        if (download) {
+            const selRes = getSelectedResources();
+            selRes.forEach((res) => {
+                window.open(
+                    `${getHost()}/api/download?path=${res.path}`,
+                    "_blank"
```

Comment:
I can't remember if this was already discussed, but do we have plans to zip up selected resources and download them in one file in the future?  This works, but if the user wants to download 10+ files, for example, at least in FireFox you get a popup asking if you want to open or save each file.

Reply:
Definitely we should add something like this for supporting bulk upload and download.